src/main/java/omero/gateway/facility/RenderFacility.java

```diff
@@ -0,0 +1,236 @@
+/*
+ *------------------------------------------------------------------------------
+ *  Copyright (C) 2024 University of Dundee. All rights reserved.
+ *
+ *
+ *  This program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2 of the License, or
+ *  (at your option) any later version.
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License along
+ *  with this program; if not, write to the Free Software Foundation, Inc.,
+ *  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ *------------------------------------------------------------------------------
+ */
+package omero.gateway.facility;
+
+import omero.ServerError;
+import omero.api.RenderingEnginePrx;
+import omero.gateway.Gateway;
+import omero.gateway.SecurityContext;
+import omero.gateway.exception.DSAccessException;
+import omero.gateway.exception.DSOutOfServiceException;
+import omero.gateway.model.ImageData;
+import omero.romio.PlaneDef;
+
+import java.awt.image.BufferedImage;
+
+public class RenderFacility extends Facility {
+    /**
+     * Creates a new instance
+     *
+     * @param gateway Reference to the {@link Gateway}
+     */
+    RenderFacility(Gateway gateway) {
+        super(gateway);
+    }
+
+    /**
+     * Get a RenderingEngine for an image.
+     *
+     * @param ctx                 The security context.
+     * @param imageId             The image ID
+     * @param initDefaultSettings Flag to create default rendering settings if the
+     *                            image doesn't have any for the current user.
+     * @return A RenderingEngine for the image
+     * @throws DSOutOfServiceException If the connection is broken, or not logged in
+     * @throws DSAccessException       If an error occurred while trying to retrieve data from OMERO
+     *                                 service.
+     */
+    public RenderingEnginePrx getRenderingEngine(SecurityContext ctx, long imageId,
+                                                 boolean initDefaultSettings)
+            throws DSOutOfServiceException, DSAccessException, ServerError {
+        return getRenderingEngine(ctx, imageId, initDefaultSettings, true);
+    }
+
+    /**
+     * Get a RenderingEngine for an image.
```

Comment:
```suggestion
     * Gets a RenderingEngine for an image.
```
To match the other methods